roles/ciao-network/README.md

```diff
@@ -1,32 +1,34 @@
-ciao-network
-============
+# ciao-network
 Ansible role to install the network node for a CIAO cluster
 
-Requirements
-------------
-None
+## Requirements
+- A previously setup of a CIAO controller node.
+  You can setup one with `ciao-controller` role
```

Comment:
[same comment](https://github.com/clearlinux/clear-config-management/pull/20/files#r70143074) applies here (point to URL of the roles)